assets/prototype/domain/eventEditor/datetimes/datetimeForm/DatetimeForm.js

```diff
@@ -61,7 +65,9 @@ const formatSecondaryField = (ticketPrice, toString = false) => {
 
 const DatetimeForm = ({ formReset, relatedTickets, tickets = [], title }) => {
 	const { id } = useContext(DateTimeContext);
-	const { description = '', name = '' } = useDatetimeItem({ id });
```

Comment:
It could be solved by `useDatetimeItem({ id }) || {}` 😄

Reply:
Removed that commit, that is out of scope of original PR.